### PR TITLE
Simplified licensing docs

### PR DESCRIPTION
**Pull request by edwarnicke**
Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

**Comment by edwarnicke** [comment]
This is an adaptation of #1 to bring things more in line with the expected norms for an Open Source project.  Most of it is boiler plate.  Particular attention should be paid to SCOPE.md as that is all original material.

File: SCOPE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact trees for artifacts.  Including but not limited to:
```
Comment (jsgf) [inline SCOPE.md:3]: We should use "graph" or "DAG" here, since "tree" is too specific.

Comment (edwarnicke) [reply SCOPE.md]: @jsgf Per the discussion yesterday I've moved the SCOPE.md to https://github.com/git-bom/spec/pull/6 .  I've also pushed an update to the PR shifting to DAGs.  Let me know what you think :)

**Comment by AevaOnline** [comment]
With the SCOPE.md pulled out, this looks good to me. Let's give it a few more days to "bake" and see if anyone else has opinions. If not, I suggest we merge by next week's meeting.

**Comment by dpp** [comment]
I'm :+1:

**Comment by AevaOnline** [comment]
Last call for comments. Will merge this tomorrow if none.

**Pull request merged**